R/zchunk_batch_land_input_4_IRR_MGMT.xml.R

```diff
@@ -28,9 +28,10 @@ module_aglu_batch_land_input_4_IRR_MGMT.xml_DISABLED <- function(command, ...) {
 
     # Produce outputs
     create_xml("land_input_4_IRR_MGMT.xml") %>%
-      add_xml_data(L2242.LN4_Logit,"LN4_Logit") %>%
+      add_logit_tables_xml(L2242.LN4_Logit,"LN4_Logit") %>%
```

Comment:
Looks like the error is occurring in this call: `add_logit_tables_xml` is expecting a `logit.type` column in `L2242.LN4_Logit` but there's not one.

@pralitp we should probably add a 
```R
  assert_that("logit.type" %in% names(data))
```
at the beginning of `add_logit_tables_xml`, yes? And what's the best/correct way for @rynacui to fix this? Thanks.

Reply:
@bpbond, good idea about the assert.  I'll open a PR.

About the missing column I think in we want to change line 67 in module_aglu_L2242.land_input_4_irr_mgmt  to (with the caveat that I haven't actually run it but I'm pretty sure):
```R
      select(one_of(c(LEVEL2_DATA_NAMES[["LN4_Logit"]], "logit.type"))) %>%
```

Reply:
Shall we use old data system behavior for
`select(one_of(c(LEVEL2_DATA_NAMES[["LN4_Logit"]], "logit.type"))) %>%`
since `logit.type` is not in the old output?

Reply:
Nope, this was a conscious decision on our part not to match the old data system about keeping the `logit.type` column.  The old / new tests will ignore that discrepancy.